autorag/utils/util.py

```diff
@@ -310,3 +310,9 @@ def sort_by_scores(row, reverse=True):
     results = sorted(zip(row['contents'], row['ids'], row['scores']), key=lambda x: x[2], reverse=reverse)
     reranked_contents, reranked_ids, reranked_scores = zip(*results)
     return list(reranked_contents), list(reranked_ids), list(reranked_scores)
+
+
+def select_top_k(row, column_names: List[str], top_k: int):
```

Comment:
The parameter `row` is actually Dataframe.
The intuitive name will be `df` instead.
The name `row` is so confusing.

Reply:
The parameter name is important, since this is util function and do not need any documentation for api. But it needs to be clear what this parameter do in the function without looking the whole code.
This is short code, but still pretty confusing if the param name is `row`